modules/linear_boltzmann_solvers/c_discrete_ordinates_curvilinear_solver/lbs_curvilinear_solver.cc

```diff
@@ -384,7 +384,7 @@ DiscreteOrdinatesCurvilinearSolver::ComputeSecondaryUnitIntegrals()
 std::shared_ptr<SweepChunk>
 DiscreteOrdinatesCurvilinearSolver::SetSweepChunk(lbs::LBSGroupset& groupset)
 {
-  auto sweep_chunk = std::make_shared<SweepChunkPWLRZ>(*grid_ptr_,
+  auto sweep_chunk = std::make_shared<SweepChunkPwlrz>(*grid_ptr_,
```

Comment:
What is the rationale for lower case `Pwlrz`? These are essentially distinct words, so in my view they should stay capitalized. Personally, I think `SweepChunkRZ` would be more appropriate. Since the discretization is not in the name for Cartesian sweep chunks, it doesn't feel like it should be for cylindrical coordinates either.

Reply:
I think it is clearer to keep things like XY, XYZ, RZ, ..., in upper case. Is this really a deviation from the standards we established?

Reply:
Yes, I agree. To me acronyms should follow the spirit of CamelCase with each distinct word getting a capital letter. I think we should revisit the change of AAH and CBC to Aah and Cbc from #130 as well.

Reply:
technically, Pwl is also an acronym so should it remain PWL ?

Reply:
In my opinion, and as the original comment suggests, yes. I think the sweep chunk naming could use some discussion. Namely, I think we should address including `PWL` in cylindrical but not Cartesian, including `RZ` but not `XYZ`, and having `AAH` and `CBC` for Cartesian, but not for cylindrical.

Reply:
100% agree. I do not like lowercase for acronyms. I prefer PWL(D), RZ, XY, AAH, etc. However, as David pointed out, this is not consistent with our coding standards. We should probably change that, I would like to broadly rename the spatial discretizations with some consistency.  Maybe something like METHOD_AGGREGATION_GEOMETRY. So, Pwlrz would become PWLD_Cellset_XYZ or PWLD_Cell_XYZ... etc. Maybe we can include this in the broader sweep organization discussion.

Reply:
I would object to the use of underscores in the class names. `RZ`, `XYZ` make sense and we can add a note into naming conventions that these are ok to use. Plus, add other exceptions you know of...